event_routing_backends/settings/common.py

```diff
@@ -173,18 +173,20 @@ def plugin_settings(settings):
         'edx.course.grade.now_failed'
     ]
 
-    settings.EVENT_BUS_TRACKING_LOGS = set(allowed_xapi_events + allowed_caliper_events)
+    allowed_events = set(allowed_xapi_events + allowed_caliper_events)
+
+    settings.EVENT_BUS_TRACKING_LOGS = allowed_events
```

Comment:
Can you document what this setting does here? It's not used anywhere else in the repo.

Reply:
is the current change enough?

Reply:
That's better, thanks